assets/tests/factories/applicationState.ts

```diff
@@ -0,0 +1,35 @@
+import { Factory } from "fishery"
+import { OpenView, State } from "../../src/state"
+import { searchPageStateFactory } from "./searchPageState"
+import { userSettingsFactory } from "./userSettings"
+
+const stateFactory = Factory.define<State>(() => ({
```

Comment:
glad to see this as a factory! Wonder if it can use the `initialState` from `../../src/state` so that if the initialState changes for some reason, the factory changes with it?

Reply:
I think I copied this from `initalState` and I didn't understand what `initalState` was when I made this, so yes!